### PR TITLE
Db compare

### PR DESCRIPTION
**Pull request by darjama**
-Added schema file
-Improved Readme file
- Refactored for Postgres database

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
+const newrelic = require('newrelic');
 const express = require('express');
 const parser = require('body-parser');
-const models = require('./models/index.js');
+const models = require('./models/pgModel.js');
+
 
 const port = 3333;
 const app = express();
+app.locals.newrelic = newrelic;
```
Comment (briansjkim) [inline server/index.js:9]: What does this do? I added newrelic too, but I didn't see this in the docs

**Review by sungjinbkim — APPROVED** [review]
everything looks good

File: server/index.js (hunk 1 above)
Comment (darjama) [reply server/index.js:9]: it was from the browser setup, not sure if it's doing anything in my setup: https://docs.newrelic.com/docs/agents/nodejs-agent/supported-features/new-relic-browser-nodejs-agent

Comment (briansjkim) [reply server/index.js:9]: Ahh right. I forgot we set up different ones. Got it

**Review by hay-0le — APPROVED** [review]

**Pull request merged**